providers/common/include/prov/providercommon.h

```diff
@@ -18,3 +18,5 @@ int cipher_capable_aes_cbc_hmac_sha1(void);
 int cipher_capable_aes_cbc_hmac_sha256(void);
 
 OSSL_FUNC_provider_get_capabilities_fn provider_get_capabilities;
+
+int provider_is_running(void);
```

Comment:
So I wonder, why is this interesting for any of our providers other than the fips provider? Would it make sense to guard this declaration with `FIPS_MODULE`, and have the alternative be a macro that just evaluates to 1?

Reply:
Ah, no, forget this, it has implications re linkage